bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -4,6 +4,37 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
+	<channel-type id="ar_destination">
+		<item-type>String</item-type>
+		<label>Route destination</label>
+		<description>Name of the destination</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ar_location" advanced="false">
+		<item-type>Location</item-type>
+		<label>Route location</label>
+		<description>Location of the destination</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="ar_distancetoarrival">
+		<item-type>Number</item-type>
```

Comment:
You could support UoM here by providing `Number:Length`. See https://www.openhab.org/docs/concepts/units-of-measurement.html#quantitytype. See for example existing channel `batteryrange`.